docs/ios/inpath/implementationSteps.md

```diff
@@ -55,11 +55,16 @@ For a full list of property descriptions, please click <a href="/docs/ios/inpath
 ## Initialise the CTContext Object with the Required Parameters
 This can be done in the view controller the SDK will be presented from.
 
+{: .important }
+The `implementationID` is needed by the SDK since it's used to fetch some configuration.<br/>
```

Comment:
I would reword this to something such as: 
The `implementationID` is needed by the SDK to fetch partner specific configuration.